### PR TITLE
Fix boolean conversion of advanced_argument_conversion option

### PR DESCRIPTION
**Pull request by mwort**
Follow-up to #33 . Mainly because `bool("False") == True`. This slipped through the tests because they use the Python booleans.

### CODE REVIEW

**Comment by percyfal** [comment]
@mwort I was also referring to #34; see comment above :)

**Comment by mwort** [comment]
@percyfal I'm confused. I cant find any comment about this PR in #35.

**Comment by percyfal** [comment]
@mwort Doh! my very very bad; I forgot to press the Review button...

File: {{cookiecutter.profile_name}}/slurm-submit.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # cookiecutter arguments
 SBATCH_DEFAULTS = """{{cookiecutter.sbatch_defaults}}"""
 CLUSTER_CONFIG = "{{cookiecutter.cluster_config}}"
-ADVANCED_ARGUMENT_CONVERSION = bool("{{cookiecutter.advanced_argument_conversion}}")
+ADVANCED_ARGUMENT_CONVERSION = {"yes": False, "no": False}["{{cookiecutter.advanced_argument_conversion}}"]
```
[the remark this answers is not in the document]
Comment (mwort) [reply {{cookiecutter.profile_name}}/slurm-submit.py]: Ooops, arggh yes of course. I think I only tested the "no" case.

**Pull request merged**